### PR TITLE
Phpcs phpdoc fixes

### LINKED ISSUE #13: Review phpcs & phpdocs fixing all warnings and errors there
While moving this repo from Travis to GHA (see #14), it has been detected that:
- The phpcs test returns a good number of warnings that should be fixed. Once done, we can enable `--max-warnings 0` in the test.
- The phpdoc test has been disabled completely. There are a bunch of missing docs here and there.

Ciao :-)

### PR DESCRIPTION
**Pull request by stronk7**
Includes 6 commits (see them for details):
- Enable phpcs and phpdoc in GHA CI workflow (and add new 404_STABLE branch tests).
- phpdoc fixes.
- http to https bulk replacement.
- more phpcs fixes.
- move harcoded text to lang strings
- yet more phpcs fixes.

Should fix #13 

Ciao :-)

IMPORTANT NOTE: This is built on top of #14 , where GHA support is added, so it must be merged after it.

### CODE REVIEW

**Comment by stronk7** [comment]
This job (in my fork, where GHA is already enabled) shows how, with the changes in this PR, both phpdoc and phpcs are now happy, 100% free from any warning / error.

https://github.com/stronk7/moodle-local_chatlogs/actions/runs/4925151869

Ciao :-)

IMPORTANT NOTE: This is built on top of https://github.com/moodlehq/moodle-local_chatlogs/pull/14 , where GHA support is added, so it must be merged after it.

**Comment by stronk7** [comment]
Friendly pong!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,13 +18,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [8.0, 8.1]
-        moodle-branch: [MOODLE_401_STABLE, MOODLE_402_STABLE, master]
-        database: [mysqli]
+        include:
```
Comment (mudrd8mz) [inline .github/workflows/ci.yml:21]: I think we can shorten this only to actual setups we use the plugin on - that is moodle.org and the integration one.

**Review by mudrd8mz — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mudrd8mz) [reply .github/workflows/ci.yml:21]: Not a blocker though,

**Comment by mudrd8mz** [comment]
> Friendly pong!

Thanks a lot!

**Comment by stronk7** [comment]
tata!

**Pull request merged**